### PR TITLE
Theme status messages not overridable

### PR DESCRIPTION
**Pull request by voidtek**
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-63

### CODE REVIEW

File: profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module [hunk 1]
```diff
@@ -494,26 +494,15 @@ function multisite_drupal_toolbox_sanitize($content, $nid) {
...
+ *
+ * This adds the alert message on all pages for the 'administrator' users.
  */
-function multisite_drupal_toolbox_messages($variables) {
-  global $user;
-  $message = "";
-  $rid = multisite_drupal_toolbox_get_rid('administrator');
-  if (variable_get('alert_message_enable', FALSE) && isset($user->roles[$rid])) {
-    $message = '<div id="alter_message" class="messages warning">' . variable_get('alert_message_body', "") . '</div>';
+function multisite_drupal_toolbox_page_alter(&$page) {
```
Comment (drupol) [inline profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module]: HOOK_INIT() FOR THE WIN !

**Comment by drupol** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Comment by gillesdeudon** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Pull request merged**